test/fixtures/es-module-loaders/loader-with-custom-condition.mjs

```diff
@@ -0,0 +1,6 @@
+export async function resolve(specifier, options, next) {
```

Comment:
Can we please use the same variable names for function arguments for all the hooks in the tests? I found it difficult to update all the tests with new signatures when I was working on my PR because the tests’ variable names were previously all over the place (`parentURL`, `parentModuleURL`, `parentModuleUrl`, etc.). I think these should just copy whatever we put in the docs; and if those change, then all the tests change.

```js
export async function resolve(specifier, context, defaultResolve) {
```

Reply:
Not worth bike-shedding here so I just used those parameter names.